.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+12.16
```

Comment:
surprised this is in the dif since its also in master.... maybe a github issue bc its not listing a conflict

Reply:
I cherry-picked @maestromac's commit so I could get my branch to build on Travis!